setup.py

```diff
@@ -32,6 +32,19 @@
             "data/html-files/fake-reports/*",
             "data/contigs/*",
             "data/references/*",
+            "data/zip_test_data/expected/folder_0/*",
+            "data/zip_test_data/expected/folder_1/subfolder_0/*",
+            "data/zip_test_data/expected/folder_1/subfolder_1/*",
+            "data/zip_test_data/expected/folder_1/subfolder_2/*",
+            "data/zip_test_data/expected/folder_2/subfolder_0/subsubfolder_0/*",
+            "data/zip_test_data/expected/folder_2/subfolder_0/subsubfolder_1/*",
+            "data/zip_test_data/expected/folder_2/subfolder_0/subsubfolder_2/*",
+            "data/zip_test_data/expected/folder_2/subfolder_1/subsubfolder_0/*",
+            "data/zip_test_data/expected/folder_2/subfolder_1/subsubfolder_1/*",
+            "data/zip_test_data/expected/folder_2/subfolder_1/subsubfolder_2/*",
+            "data/zip_test_data/expected/folder_2/subfolder_2/subsubfolder_0/*",
+            "data/zip_test_data/expected/folder_2/subfolder_2/subsubfolder_1/*",
+            "data/zip_test_data/expected/folder_2/subfolder_2/subsubfolder_2/*",
```

Comment:
You could replace it with (you'll only really know if it worked properly after the CI is finished):
```suggestion
            "data/zip_test_data/expected/*",
            "data/zip_test_data/expected/*/*",
            "data/zip_test_data/expected/*/*/*",
            "data/zip_test_data/expected/*/*/*/*",
```